src/bindings/python/src/pyopenvino/core/compiled_model.cpp

```diff
@@ -40,17 +42,39 @@ void regclass_CompiledModel(py::module m) {
 
     cls.def("export_model", &ov::CompiledModel::export_model, py::arg("model_stream"));
 
+    cls.def(
+        "set_property",
+        [](ov::CompiledModel& self, const std::map<std::string, py::object>& properties) {
+            std::map<std::string, PyAny> properties_to_cpp;
+            for (const auto& property : properties) {
+                properties_to_cpp[property.first] = PyAny(property.second);
+            }
+            self.set_property({properties_to_cpp.begin(), properties_to_cpp.end()});
+        },
+        py::arg("properties"));
+
+    // todo: remove after Accuracy Checker migration to set/get_property API
     cls.def(
         "get_config",
```

Comment:
Do we have a plan to remove `get_confg` methods?

Reply:
Yes, as OMZ migrates to the new configuration API (77081 will be completed), I'll remove `set/get_config` and `get_metric` methods.